systemtest/rum_test.go

```diff
@@ -36,8 +38,63 @@ import (
 	"github.com/elastic/apm-server/systemtest/estest"
 	"github.com/elastic/apm-tools/pkg/approvaltest"
 	"github.com/elastic/apm-tools/pkg/espoll"
+	"github.com/elastic/go-elasticsearch/v8/esapi"
 )
 
+func TestRUMGeoIpTags(t *testing.T) {
+	systemtest.CleanupElasticsearch(t)
+
+	// Ensure the geo-database does not get downloaded.
+	b, err := json.Marshal(map[string]interface{}{
+		"persistent": map[string]interface{}{
+			"ingest.geoip.downloader.enabled": false,
+		},
+	})
+	require.NoError(t, err)
+
+	r := esapi.ClusterPutSettingsRequest{
+		Body: bytes.NewReader(b),
+	}
```

Comment:
Why would this be set to `false`? The goal is to trigger the geoip downloader and have the relevant fields in the ingested events geoip enriched.
The `tags` field would only be set if an error happened during this download or enrichment, so we want to check that `tags` is empty.